olive/engine/engine.py

```diff
@@ -637,3 +656,17 @@ def _evaluate_model(self, model: OliveModel, model_id: str, evaluator: OliveEval
             ),
         )
         return signal
+
+    def _get_top_ranked_nodes(self, metrics: List[Metric], footprint: Footprint, k: int) -> List[FootprintNode]:
+        metric_priority = [metric.name for metric in sorted(metrics, key=lambda x: x.priority_rank)]
+        footprint_node_list = footprint.nodes.values()
```

Comment:
![image](https://user-images.githubusercontent.com/13343117/231978939-afefe686-f97a-42d6-8eab-73f3bfe42df7.png)

In some cases, the input torch model is under the origin pareto frontier like above. If we adapt the accuracy as top priority metric. Then the torch model will be first to be packaged. 

Sorry for the confusion for output torch model in previous PR.
Do you mind to remove torch_model in this PR?